### PR TITLE
remove velocity based haptics on interactables

### PR DESCRIPTION
**Pull request by InfiniteLee**
instead, pulse when grabbing and releasing. Also enable haptics for the cursor.

### CODE REVIEW

File: src/hub.html [hunk 1]
```diff
@@ -329,6 +329,7 @@
                   missOpacity: 0.1;
                   curveShootingSpeed: 12;"
               haptic-feedback
+              event-repeater="events: haptic_pulse; eventSource: #cursor"
```
Comment (brianpeiris) [inline src/hub.html]: Add this to the left controller as well?

Comment (InfiniteLee) [reply src/hub.html]: Actually that brings up a good point. On 6-dof, having this on the left largely doesn't matter as your cursor is only ever on the right hand. However, that is not the case for 3dof devices that may be left-handed only. That said, haptics only exist on 6-dof right now, and so this should be sufficient. (If we did add it to the left anyways, both controllers would get a haptic pulse when the cursor triggered one).

Comment (netpro2k) [reply src/hub.html]: We currently only support right hand on 3dof anyway, so this sint an issue, though we obviously need to address the larger issue of handedness throughout.

Comment (InfiniteLee) [reply src/hub.html]: >We currently only support right hand on 3dof anyway
Is not true, we respect if you have your controller set to left-handed on go, gearvr and daydream

**Pull request merged**